client/ayon_deadline/plugins/publish/global/submit_publish_job.py

```diff
@@ -501,8 +511,20 @@ def _get_publish_folder(self, anatomy, template_data,
             "name": product_name,
             "type": product_type,
         }
+        output = None
+        try:
+            render_dir_template = anatomy.get_template_item(
+                "publish", template_name, "directory"
+            )
+            output = (
+                render_dir_template
+                .format_strict(template_data)
+                .replace("\\", "/")
+            )
```

Comment:
Shouldn't this be outside of the try except?
```suggestion
        render_dir_template = anatomy.get_template_item(
            "publish", template_name, "directory"
        )
        output = None
        try:
            output = (
                render_dir_template
                .format_strict(template_data)
                .replace("\\", "/")
            )
```

Also, the `\\` to `/` replacement seems new in this PR? Any reason?

Reply:
Oh, I did want to capture `KeyError` too, but then realized it is probably not good idea and didn't change it...